src/server/FileParser/ProcessFiles.ts

```diff
@@ -129,112 +131,118 @@ function processFile(sourceFile: SourceFile) {
   return getCodeInformation(sourceFile);
 }
 
-function processClassDeclaration(node: Node) {
+interface IProcessedClassDeclaration {
```

Comment:
Your pref to have that one below `function processFile`?

Reply:
I'd actually like to have it on top of the file :) It was just placed there as I drilled downward and that was the first time it became apparent I needed it. I'll push a new change.

Reply:
I'm fine with having rough changes.
An initial comment would be nice to indicate that :)